src/Listener/ConsoleListener.php

```diff
@@ -16,14 +16,14 @@
 class ConsoleListener
 {
     /** @var EventDispatcherInterface */
-    protected $eventDispatcher = null;
+    protected $eventDispatcher;
```

Comment:
You should update line 67 to check with isset

Reply:
To be honest, I trusted blindly the cs-fixer. Thank you.